libs/ui/src/lib/line-chart/line-chart.component.ts

```diff
@@ -209,6 +210,10 @@ export class LineChartComponent implements AfterViewInit, OnChanges, OnDestroy {
               tooltip: this.getTooltipPluginConfiguration(),
               verticalHoverLine: {
                 color: `rgba(${getTextColor(this.colorScheme)}, 0.1)`
+              },
+              decimation: {
+                enabled: this.dataDecimation,
+                algorithm: 'min-max'
```

Comment:
According to the [documentation](https://www.chartjs.org/docs/latest/configuration/decimation.html), `min-max` is the default value, so not needed at all.